### DISCUSSION TITLE
Free Trial available for ChistaData Cloud?

### DISCUSSION THREAD

**Discussion by emoido** [ChistaDATA/Cloud#5] [❓FAQ] [answered]
Free Trial available for ChistaData Cloud?

**Comment by TheraniAA** [comment, marked as answer]
ChistaData Cloud offers a complimentary trial period that spans 90 days. Once the trial period elapses, users can either switch to a paid subscription or allow the cluster to go into sleep mode. If the cluster remains inactive for more than 90 days, it will be automatically removed. For those who want to test the waters before making a commitment, ChistaData Cloud welcomes interested users to create a free account and take advantage of the trial period. If you’re ready to explore this service, follow this link to sign up: https://chistadata.io/
